test/Campaign.test.js

```diff
@@ -0,0 +1,85 @@
+const assert = require("assert");
+const CampaignFactory = artifacts.require("CampaignFactory");
+const Campaign = artifacts.require("Campaign");
+
+contract("CampaignFactory", async accounts => {
+  it("can deploy a Campaign", async () => {
+    let campaignAddress
+    let factory = await CampaignFactory.deployed()
+
+    await factory.createCampaign(200, {from: accounts[0]});
```

Comment:
Where is accounts[0] coming from?

Reply:
It's coming from line 9. Truffle gives us a `contract` block we can use for testing that generates 10 mock accounts with 100 eth for each contract block.